java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/project/BrokenReferencesModel.java

```diff
@@ -116,50 +116,49 @@ void refresh() {
      */
     void refresh(boolean fire) {
         AtomicBoolean changed = new AtomicBoolean(false);
```

Comment:
Nit - the `AtomicBoolean` could probably be changed to ordinary `boolean`, and moved inside the request, next to `int size;`.

Reply:
Recommitted as a4edd994da94